screener/views.py

```
@@ -746,3 +749,41 @@ def get(self, request, white_label):
 
         return Response({"generic": generic, "partners": partners})
+
+
+class RemImpactView(views.APIView):
+    """
+    Proxies a request to the Rewiring America REM /api/v1/rem/address endpoint
+    and returns only the total cost delta (bill impact) and total emissions delta
+    that the frontend needs to render the Calculate Impact results view.
+    """
+
+    permission_classes = [permissions.AllowAny]
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)